packages/grafana-data/src/datetime/moment_wrapper.ts

```diff
@@ -5,7 +5,7 @@ export interface DateTimeBuiltinFormat {
   __momentBuiltinFormatBrand: any;
 }
 export const ISO_8601: DateTimeBuiltinFormat = moment.ISO_8601;
-export type DateTimeInput = Date | string | number | Array<string | number> | DateTime; // null | undefined
+export type DateTimeInput = Date | string | number | Array<string | number> | DateTime | null; // | undefined;
```

Comment:
Had to uncomment this to allow null values, no idea why it was commented out in the first place.

Reply:
Yeah, no idea neither, but it should be OK